packages/dds/tree/src/shared-tree/sharedTree.ts

```diff
@@ -108,30 +110,26 @@ export interface SharedTreeContentSnapshot {
 }
 
 /**
- * {@link ITree} extended with some non-public APIs.
+ * Information about a Fluid channel.
+ * @privateRemarks
+ * This is distinct from {@link IChannel} as it omits the APIs used by the runtime to manage the channel and instead only has things which are useful (and safe) to expose to users of the channel.
  * @internal
  */
-export interface ITreeInternal extends ISharedObject, ITree {
-	/**
-	 * Exports root in the same format as {@link TreeAlpha.(exportVerbose:1)} using stored keys.
-	 * @privateRemarks
-	 * TODO:
-	 * This should probably get promoted to a public API on ITree eventually.
-	 */
-	exportVerbose(): VerboseTree | undefined;
+export type IChannelView = Pick<IChannel, "id" | "attributes" | "isAttached">;
 
-	/**
-	 * Exports the SimpleTreeSchema that is stored in the tree, using stored keys for object fields.
-	 * @remarks
-	 * To get the schema using property keys, use {@link getSimpleSchema} on the view schema.
-	 */
-	exportSimpleSchema(): SimpleTreeSchema;
-}
+/**
+ * {@link ITree} extended with some non-public APIs.
+ * @internal
+ */
+export interface ITreeInternal extends IChannelView, ITreeAlpha {}
 
 /**
  * {@link ITreeInternal} extended with some non-exported APIs.
+ * @remarks
```

Comment:
Nit: should we add a `@privateRemarks` note that this should not be package-exported? Seems self-evident to me, but in my experience that hasn't been sufficient to stop over-exposure in the past :/

Reply:
API extractor will complain if someone tries to package export it as it references a large ~30 non-exported types transitively (I know as I made this non exported in https://github.com/microsoft/FluidFramework/pull/23256/files#diff-975a282386c223fc881ab3686189ebe46f40fc98bc3942d6c70fad2a5ca044f5 ). I suspect that will deter people sufficiently.